src/main/java/de/retest/web/selenium/UnbreakableDriver.java

```diff
@@ -53,8 +54,8 @@
 	 *            We use RemoteWebDriver instead of WebDriver, because we need some of the other interfaces
```

Comment:
The Javadoc in line 53 needs to be adapted to fix the build:

```suggestion
	 * @param wrappedDriver
```

Reply:
Done